src/client/theme-default/components/VPLocalSearchBox.vue

```diff
@@ -299,26 +299,40 @@ function scrollToSelectedResult() {
   })
 }
 
-onKeyStroke('ArrowUp', (event) => {
-  event.preventDefault()
-  selectedIndex.value--
-  if (selectedIndex.value < 0) {
-    selectedIndex.value = results.value.length - 1
-  }
+function goRelativeItem(offset: number) {
+  const resNum = results.value.length
+  selectedIndex.value = (selectedIndex.value + resNum + (offset % resNum)) % resNum
```

Comment:
when `resNum == 0`, the `selectedIndex.value` will be `NAN`, but this won't crash the script, so I don't add more judgement